### DISCUSSION TITLE
How to style/hide the wire:navigate loading indicators

### DISCUSSION THREAD

**Discussion by jacksleight** [livewire/livewire#5963] [1. Help] [answered]
Is there a way to style or hide the wire:navigate loading indicators? eg. to change the colour or just have the bar and no spinner?

I had a look in the DOM and I guess I can just target the `bar` and `spinner` classes,  but I was wondering if there was a more official way? Also it might be useful to be able to customise these classes, in case they conflict with others in the app.

**Comment by nleibbrandt** [comment, marked as answer]
I'm pretty sure there will be a better solution in the future, for now please check https://github.com/livewire/livewire/discussions/5830

**Comment by joshhanley** [comment]
@jacksleight you should be able to disable the navigation bar in the next beta release, see PR #5874.
